python/lsst/ts/vent/controller/dispatcher.py

```diff
@@ -142,11 +175,25 @@ async def read_and_dispatch(self) -> None:
                 )
             )
 
-    async def close_vent_gate(self, gate: int) -> None:
-        self.controller.vent_close(gate)
+    async def close_vent_gate(
+        self, gate0: int, gate1: int, gate2: int, gate3: int
+    ) -> None:
+        for gate in (gate0, gate1, gate2, gate3):
+            if gate >= 0 and gate <= 3:
```

Comment:
This gives me the impression the value of `gate` could be a enum instead. Then you could check it agains valid enum values. But I am not sure I fully understand what it going on here.

Reply:
Yeah, were you involved with that discussion on Slack? Basically  ventOpenGate and ventCloseGate take four ints, and we were loathe to change the CSC if we could avoid that. The result of the discussion was that each gate number passed (0 through 3) would be opened/closed, and -1 would be used as a placeholder.